packages/nimble-components/src/wafer-map/specs/features/change-highlight.md

```diff
@@ -0,0 +1,56 @@
+# Nimble Wafer Map Die Highlighting
+
+## Problem Statement
+
+The wafer map receives an array of strings as the values that will be highlighted.
+
+```
```

Comment:
make sure here and in the examples below to mark the code fence with `ts` so that it has syntax highlighting, i.e.
````
```ts
@observable public highlightedValues: string[] = [];
```
````

which renders as
```ts
@observable public highlightedValues: string[] = [];
```